src/bin/console/console.cpp

```diff
@@ -760,8 +796,8 @@ char** Console::_command_completion(const char* text, int start, int end) {
   // Choose completion function depending on the input. If it starts with "generate",
   // suggest TPC-C tablenames for completion.
   const std::string& first_word = tokens.at(0);
-  if (first_word == "generate") {
-    // Completion only for two words, "generate", and the TABLENAME
+  if (first_word == "gentpch") {
```

Comment:
Personally, I would like `generate tpc[ch]` better. Newcomers to the database benchmarking world will have no idea what `gentpch` might mean (or how to pronounce it :D )

Reply:
I turned it into `generate_tp{ch}`, is that okay? `generate tpch` with a fixed singlespace between "generate" and "tpch" would work as well, but that makes it look like we have some kind of sub-command system and we don't have that

Reply:
ok